### PR TITLE
Sprint 6

### PR DESCRIPTION
**Pull request by AYogore**
Score UI implemented
Infinte jump and shoot limited with bools and firerate

### CODE REVIEW

File: KaijuKiller/Assets/Scripts/ScoreUI.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using TMPro;
+using UnityEngine.SocialPlatforms.Impl;
+
```
Comment (Kobakat) [inline KaijuKiller/Assets/Scripts/ScoreUI.cs:7]: Nice approach to this class. I like the encapsulation of score

File: KaijuKiller/Assets/Scripts/PlayerMove.cs [hunk 2]
```diff
@@ -76,6 +78,11 @@ void Update()
 
     }
```
Comment (Kobakat) [inline KaijuKiller/Assets/Scripts/PlayerMove.cs:80]: There is potential for this to break things but given the context of your game its probably ok for now

**Review by Kobakat — COMMENTED** [review]
Nice work, if I had to suggest what was next, I would try and make the jump animation look less choppy.

File: KaijuKiller/Assets/Scripts/ScoreUI.cs [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+public class ScoreUI : MonoBehaviour
+{
+    
+    private static int score;
+
+    [SerializeField]
+    public TextMeshProUGUI scoreText;
```
Comment (dantogno) [inline KaijuKiller/Assets/Scripts/ScoreUI.cs:14]: don't need to serialize public fields. But maybe it should be private?

**Pull request merged**